blocks.js

```diff
@@ -7529,7 +7529,6 @@ InputSlotMorph.prototype.mouseDownLeft = function (pos) {
         this.escalateEvent('mouseDownLeft', pos);
     } else {
         this.contents().edit();
-        this.contents().selectAll();
```

Comment:
Hi Jens!

Sure, the issue was that this broke the shitf+click selection feature. You can just try to re-add this line and then play with shift click in, for example, a JS block.